content/en/synthetics/_index.md

```diff
@@ -22,53 +22,57 @@ further_reading:
 
-### Set up API tests and multistep API tests
+## Set up API tests and multistep API tests
 
-API tests allow you to launch [single][1] or [chained][2] requests to perform verifications on your key systems at various network levels: `HTTP`, `TCP`, `SSL`, `DNS`, and `ICMP`. Create your first [HTTP test][3], [TCP test][4], [SSL test][5], [DNS test][6], or [ICMP test][7] to get started with API and network monitoring.
+API tests allow you to launch [single][1] or [chained][2] requests to perform verifications on your key systems at various network levels: `HTTP`, `TCP`, `SSL`, `DNS`, and `ICMP`. 
 
 {{< img src="synthetics/api_test.png" alt="API tests"  style="width:100%;">}}
 
-### Record browser tests
+## Record browser tests
 
-Record end-to-end tests to monitor how your customers experience your webpages from around the world using [Synthetic browser tests][8].
+Use [Synthetic browser tests][3] to record end-to-end tests and monitor how your customers experience your webpages from around the world.
 
 {{< img src="synthetics/browser_test.gif" alt="Browser tests"  style="width:100%;">}}
 
-### Launch private locations
+## Launch private locations
 
-Use [Synthetic private locations][9] to monitor internal APIs and websites or to create custom locations in areas that are mission-critical to your business.
+Use [Synthetic private locations][4] to monitor internal APIs and websites or create custom locations in areas that are mission-critical to your business.
 
 {{< img src="synthetics/private_locations.png" alt="Private locations"  style="width:100%;">}}
 
-### Run tests with your integration and deployment processes
+## Run tests with your integration and deployment processes
 
-Leverage your Synthetic tests as [canaries][10] or run them directly within your [CI pipelines][10] to start shipping without fearing faulty code might impact your customers experience.
+Leverage your Synthetic tests as [canaries][5] or run them directly within your [CI pipelines][5] to start shipping without fear that faulty code may impact your customers' experience.
 
  {{< img src="synthetics/ci.png" alt="CI tests"  style="width:100%;">}}
 
-### Connect data and traces
+## Connect data and traces
 
-Use the [out of the box integration between Synthetic tests and APM traces][11] to find the root cause of failures across frontend, network and backend requests.
+Use the [out of the box integration between Synthetic tests and APM traces][6] to find the root cause of failures across frontend, network, and backend requests.
 
 {{< img src="synthetics/synthetics_traces.gif" alt="Synthetic Monitoring" style="width:100%;">}}
 
+## Ready to start?
+
+See [Getting Started with Synthetic Monitoring][7] for instructions on creating your first Synthetic test and monitoring your web applications. Then, start exploring API tests with [HTTP test][8], [SSL test][9], [TCP test][10], [DNS test][11], or [ICMP test][12].
```

Comment:
Why put API tests forward more than Browser tests, private locations, CI, and integration with APM with the second sentence? How about just having the first sentence and mention that users will find info on how to get started with private locations too?